portfolio.adobe.com.stage.json

```diff
@@ -0,0 +1,40 @@
+{
+  "providerId":"portfolio.adobe.com",
```

Comment:
this should be changed to the stage url like matt g suggested, right?

Reply:
I don't think so. From their docs:

>providerId:
>
>The unique identifier of the Service Provider that created this template. This is used in the URLs to identify the Service Provider. To ensure non-coordinated uniqueness, this should be the domain name of the Service Provider (e.g. exampleservice.domainconnect.org).

It seems like `providerId` is just an identifier, and a way to link users to the website of the provider that they are choosing to connect to in addition to the display name. It's not used for any sort of whitelisting or anything. Domain Connect does not need to know what the URL base is for stage or prod Portfolio sites, it just needs access to the information needed to process the connection request. When we ask for Domain Connect info, we send the name of the template we want them to use – `portfolio.adobe.com.prod.json` or `portfolio.adobe.com.stage.json` – and that's how they choose which IP addresses to use for the URL being set up.